with-proxy/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2'
+
+services:
+
+  nginx:
+    image: nginx
+    volumes:
+      - ./nginx:/etc/nginx:ro
+    networks:
+      - private
+
+  squid:
+    image: babim/squid
+    networks:
+      - public
+      - private
+
+  vector:
+    image: timberio/vector:nightly-alpine
+    volumes:
+      - ./vector/vector.toml:/etc/vector/vector.toml:ro
+      - ./logs:/tmp/logs
```

Comment:
Do we need this directory mounted? I'm not seeing anything using it.

Reply:
ah good point, a copy/paste that went too fast...